Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Fleet/Registration.hs

```diff
@@ -127,7 +136,32 @@ fleetOwnerRegister mbRequestorId req = do
       let req' = Image.ImageValidateRequest {imageType = DVC.BusinessLicense, image = businessLicenseImage, rcNumber = Nothing, validationStatus = Nothing, workflowTransactionId = Nothing, vehicleCategory = Nothing, sdkFailureReason = Nothing}
       image <- Image.validateImage True (req.personId, person.merchantId, person.merchantOperatingCityId) req'
       QFOI.updateBusinessLicenseImage (Just image.imageId.getId) req.personId
-  return Success
+  enabled <- enableFleetIfPossible
+  return $ FleetOwnerUpdateRes enabled
```

Comment:
Lets update `enabled` on bpp side also in this function `updateFleetOwnerInfo`
Because this field used in `checkFleetOwnerVerification` and fleet apis could fail without this update